lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1451,6 +1451,8 @@ namespace Js
 
     RecyclableObject* JavascriptProxy::GetPrototypeSpecial()
     {
+        PROBE_STACK(GetScriptContext(), Js::Constants::MinStackDefault);
```

Comment:
discussed offline, Akrosh will check all proxy apis and make sure we have stack probe where needed